api/nakadi-event-bus-api.yaml

```diff
@@ -888,6 +888,7 @@ definitions:
               Timestamp of creation of the Event generated by the producer.
             type: string
             format: date-time
+            example: '1996-12-19T16:39:57-08:00'
```

Comment:
I would be very careful with time parsing. There are so many corner cases, that is is better to work with special library like JODA

Have loom here:https://rafaelsantosbra.wordpress.com/tag/rfc-3339/